### DISCUSSION TITLE
select2_from_ajax with entity false doesn't show the stored value

### DISCUSSION THREAD

**Discussion by miquelangeld** [Laravel-Backpack/community-forum#384] [Q&A (Help)]
I'm using fields with select2_from_ajax to get options from external DB (with entity attribute = false) and it's working fine, but when I edit an entry this field doesn't show the DB value. The select appear empty, but the value are in DB fot that field. Any idea?

**Comment by pxpm** [comment]
Hey @miquelangeld 
It could be `placeholder` or the `default` ?

**Reply by miquelangeld** [reply]
I don't use any of them. This issue is about the same fields that we discussed [here](https://github.com/Laravel-Backpack/CRUD/issues/4907) 

This is the field:
$this->crud->addField([
            'label'       => "Booking", 
            'type'        => "select2_from_ajax",
            'name'        => 'booking', 
            'entity'      => false, 
            'attribute'   => "booking", 
            'model'		  => "App\Models\CustomerCase",
            'data_source' => url('/admin/bookings/ajax-bookings-avalon-options')
        ]);

**Reply by pxpm** [reply]
Is there a column on database called `booking` ? 

If not you can always create an acessor in your model:
```php
public function getBookingAttribute() {
...
}
```

**Reply by miquelangeld** [reply]
Yes, exist.
